lib/vault/docker-compose.yml

```diff
@@ -33,10 +28,40 @@ services:
         - "traefik.docker.lbswarm=true"
         - "traefik.http.routers.vault.rule=Host(`vault.${CANDIG_DOMAIN}`)"
         - "traefik.http.services.vault.loadbalancer.server.port=${VAULT_SERVICE_PORT}"
+    secrets:
+      - source: vault-s3-token
+        target: vault-s3-token
     logging: *default-logging
-    command: server -config=/vault/config/vault-config.json
+    command: vault server -config=/vault/config/vault-config.json
     healthcheck:
       test: [ "CMD", "curl", "-f", "http://0.0.0.0:${VAULT_SERVICE_PORT}/ui/" ]
       interval: 30s
       timeout: 20s
       retries: 3
+  vault-runner:
```

Comment:
Why is vault runner needed?

Reply:
That being said, the cronjob doesn't seem to work correctly. The token can be manually refreshed via the create_token.sh script, though.